### PR TITLE
Create ApyChart component

### PR DESCRIPTION
**Pull request by therealemjy**
- update Rechart to 2.1.9
- add `tiny` variant to theme (recently added to designs: https://www.figma.com/file/GYJwWWLbC4B8DrJOPcETz3/Venus-2.0?node-id=7%3A203)
- update theme type to make it stricter
- Create `ApyChart` component
Screen Shot 2022-05-10 at 16 32 51
Screen Shot 2022-05-10 at 16 32 54

### CODE REVIEW

File: src/components/v2/ApyChart/formatToReadableDate.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import moment from 'moment';
```
Comment (coreyar) [inline src/components/v2/ApyChart/formatToReadableDate.ts]: `moment` isn't recommended anymore. We should look at using `luxon` or `date-fns`

File: src/components/v2/ApyChart/index.tsx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+                        }),
+                      }}
+                    />
+                  </div>
+                </div>
+              ) : (
+                // Although the type definition of "content" allows an undefined
+                // value to be returned, it actually creates a runtime error if
+                // we do so. For that reason we return a fragment instead
+                <></>
```
Comment (coreyar) [inline src/components/v2/ApyChart/index.tsx]: null should be returned in place of rendering `undefined`. We can also remove the comment if we return null

File: src/components/v2/ApyChart/formatToReadableDate.ts (hunk 1 above)
Comment (therealemjy) [reply src/components/v2/ApyChart/formatToReadableDate.ts]: Works for me 👍

File: src/components/v2/ApyChart/index.tsx (hunk 2 above)
Comment (therealemjy) [reply src/components/v2/ApyChart/index.tsx]: That's much nicer 👍

**Review by coreyar — APPROVED** [review]

**Pull request merged**